### DISCUSSION TITLE
Who recently won their first ⛳🏌️ Masters?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#610] [Galaxy Trivia] [answered]
**Bonus:** How old is the golfer?

**Comment by peterbaccaro** [comment]
**Jon Rahm** recently won his first Masters at the age of **28**.

**Comment by thuongtruong1009** [comment, marked as answer]
As of my knowledge cutoff of September 2021, Hideki Matsuyama won his first Masters tournament in April 2021. He was 29 years old at the time. However, please note that as an AI language model, I do not have real-time access to current events beyond my knowledge cutoff date.
